Layouts/default.php

```diff
@@ -11,5 +11,7 @@
     <?= $content ?>
     <header><?= $this -> renderSubView("header") ?></header>
     <aside><?= $this -> renderSubView("sideMenu") ?></aside>
+    <main><?= $this -> renderSubView("main")?></main>
+    <footer><?= $this -> renderSubView("footer") ?></footer>
```

Comment:
Een andere oplossing zou kunnen zijn dat je de footer in de main zet dan hoef je hem niet positioneren en kan het width 100% zijn, Maar dit is ook een prima oplossing